app/src/main/java/com/inveitix/android/compass/constants/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.inveitix.android.compass.constants;
+
+public class Constants {
```

Comment:
Most of these constants are used only once and have a specific use. For these cases, best leave the constant within it's class.